src/components/movie-details/movie-details.jsx

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {movieShape} from "../shapes";
+
+const MovieDetails = (props) => {
+  const {movie} = props;
+  const {
+    actors,
+    bigPoster,
+    descriptions,
```

Comment:
обычно в практике описание переводится в единсвенном числе 
```suggestion
    description,
```